code/drasil-example/README.md

```diff
@@ -1,57 +1,57 @@
 --------------------------------------------------
 ### Summary of Folder Structure and File Contents
-Last updated: July 22, 2021
+Last updated: July 31, 2024
 --------------------------------------------------
 
 This folder contains our working examples (implementations in Drasil of the previously manually created case studies).
 
 **dblpend**
   - Contains the current state of the Double Pendulum case study implemented in Drasil
   - This is a re-creation of the original, manually-created version 
-    in the [dblpend](https://github.com/Zhang-Zhi-ZZ/CAS741Project) Double Pendulum example.
+    in the [dblpend](https://github.com/Zhang-Zhi-ZZ/CAS741Project/blob/master/Double%20Pendulum/docs/SRS/SRS.pdf) Double Pendulum example.
```

Comment:
I noticed the all links direct to the PDF SRS of the manually created case studies. @smiths correct me if I'm wrong, but it makes more sense to link to the case study directory instead. For this particular instance, it should be:

<https://github.com/Zhang-Zhi-ZZ/CAS741Project/blob/master/Double%20Pendulum>

Reply:
Yeah, I updated the links more specifically based on the suggestions from https://github.com/JacquesCarette/Drasil/pull/3682.

Reply:
@Xinlu-Y I agree with the feedback from @BilalM04.  I should have mentioned it previously.  Let's link to the case study directory, rather than the SRS for each example.

Reply:
Ok, I will modify these links as soon as possible.